packages/webdriverio/tests/commands/element/shadow$.test.js

```diff
@@ -9,15 +17,8 @@ describe('shadow$', () => {
             }
         })
         const el = await browser.$('#foo')
-        // this is wacky, but I'm not all that familiar with jest.
-        // if I tried to spyOn the el directly, I get an error:
-        // 'Cannot assign to read only property'
-        const clone = Object.assign({}, el)
-        clone.shadow$ = el.shadow$
-        clone.$ = el.$
-        const spy = jest.spyOn(clone,'$')
-        clone.shadow$('#shadowfoo')
-        expect(spy).toHaveBeenCalledWith(expect.any(Function))
-        spy.mockClear()
+
+        await el.shadow$('#shadowfoo')
```

Comment:
This is currently broken due to the command not being awaited. This PR just exposes the broken UT (and cleans it up).